phoenix-core/src/test/java/org/apache/phoenix/compile/QueryCompilerTest.java

```diff
@@ -1453,7 +1453,7 @@ public void testInvalidPrimaryKeyDecl() throws Exception {
             }
         }
     }
-    
+
```

Comment:
@lomoree It's nice to avoid these trailing spaces, but most of the time, we need to avoid such changes in our patch, especially for 'calcite' branch at the moment, for it would make future merge to master more difficult.